Dockerfile

```diff
@@ -39,5 +39,6 @@ LABEL revision ${GIT_REVISION}
 EXPOSE 8080
 ENV HOST=0.0.0.0
 COPY --from=builder /src/target/release/migration /migration
+COPY --from=builder /src/target/release/migrate_to /migrate_to
```

Comment:
Do we still need `migration` in the Dockerfile if we intend to use `migrate_to` for DivviUp ops? Are there users of the Dockerfile that use `migration`?

Reply:
I was approaching it as a 'nice to have'. It may still be useful in ops contexts, e.g. the `migration up` to go to latest could be useful in some e2e tests, but none that are in place right now. By YAGNI, it can be removed.